### PR TITLE
changing dot size in plot depending if new submission #75

### PR DESCRIPTION
**Pull request by wolski**

### CODE REVIEW

File: webinterface/_base.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def _preface(self):
             unsafe_allow_html=True,
         )
 
-    @abstractmethod
+    @abstractmethod
```
Comment (enryH) [inline webinterface/_base.py:39]: can you install the black formatting plugin in VSCode?

**Comment by enryH** [comment]
So both your changes are due to formatting. We use black for formatting the code and it can be installed as a plugin in VSCode: https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter

File: proteobench/modules/dda_quant/module.py [hunk 2]
```diff
@@ -168,7 +167,11 @@ def add_current_data_point(self, all_datapoints, current_datapoint):
         """Add current data point to all data points and load them from file if empty. TODO: Not clear why is the df transposed here."""
         if not isinstance(all_datapoints, pd.DataFrame):
             all_datapoints = pd.read_json(DDA_QUANT_RESULTS_PATH)
+        all_datapoints["old_new"] = "old"
+        
         all_datapoints = all_datapoints.T
+        
+        current_datapoint["old_new"] = "new"
```
Comment (enryH) [inline proteobench/modules/dda_quant/module.py:174]: Alternatively you could remember the index and use `df.loc[idx_new]`

File: proteobench/modules/dda_quant/plot.py [hunk 3]
```diff
@@ -75,12 +75,25 @@ def plot_metric(self, benchmark_metrics_df: pd.DataFrame) -> go.Figure:
 
         # Add hover text
         hover_texts = [
-            f"Search Engine: {benchmark_metrics_df.search_engine[idx]} {benchmark_metrics_df.software_version[idx]}<br>FDR psm: {benchmark_metrics_df.fdr_psm[idx]}<br>FDR Peptide: {benchmark_metrics_df.fdr_peptide[idx]}<br>FRD Protein: {benchmark_metrics_df.fdr_protein[idx]}<br>MBR: {benchmark_metrics_df.MBR[idx]}<br>Precursor Tolerance: {benchmark_metrics_df.precursor_tol[idx]} {benchmark_metrics_df.precursor_tol_unit[idx]}<br>Fragment Tolerance: {benchmark_metrics_df.fragment_tol_unit[idx]}<br>Enzyme: {benchmark_metrics_df.enzyme_name[idx]} <br>Missed Cleavages: {benchmark_metrics_df.missed_cleavages[idx]}<br>Min peptide length: {benchmark_metrics_df.min_pep_length[idx]}<br>Max peptide length: {benchmark_metrics_df.max_pep_length[idx]}"
+            f"Search Engine: {benchmark_metrics_df.search_engine[idx]} {benchmark_metrics_df.software_version[idx]}<br>" +
```
Comment (enryH) [inline proteobench/modules/dda_quant/plot.py:78]: you don't need the plus:)

File: proteobench/modules/dda_quant/plot.py [hunk 4]
```diff
@@ -75,26 +77,41 @@ def plot_metric(self, benchmark_metrics_df: pd.DataFrame) -> go.Figure:
 
         # Add hover text
         hover_texts = [
-            f"Search Engine: {benchmark_metrics_df.search_engine[idx]} {benchmark_metrics_df.software_version[idx]}<br>FDR psm: {benchmark_metrics_df.fdr_psm[idx]}<br>FDR Peptide: {benchmark_metrics_df.fdr_peptide[idx]}<br>FRD Protein: {benchmark_metrics_df.fdr_protein[idx]}<br>MBR: {benchmark_metrics_df.MBR[idx]}<br>Precursor Tolerance: {benchmark_metrics_df.precursor_tol[idx]} {benchmark_metrics_df.precursor_tol_unit[idx]}<br>Fragment Tolerance: {benchmark_metrics_df.fragment_tol_unit[idx]}<br>Enzyme: {benchmark_metrics_df.enzyme_name[idx]} <br>Missed Cleavages: {benchmark_metrics_df.missed_cleavages[idx]}<br>Min peptide length: {benchmark_metrics_df.min_pep_length[idx]}<br>Max peptide length: {benchmark_metrics_df.max_pep_length[idx]}"
+            f"Search Engine: {benchmark_metrics_df.search_engine[idx]} {benchmark_metrics_df.software_version[idx]}<br>"
+            + f"FDR psm: {benchmark_metrics_df.fdr_psm[idx]}<br>"
+            + f"FDR Peptide: {benchmark_metrics_df.fdr_peptide[idx]}<br>"
+            + f"FRD Protein: {benchmark_metrics_df.fdr_protein[idx]}<br>"
```
Comment (mlocardpaulet) [inline proteobench/modules/dda_quant/plot.py:83]:
```suggestion
            + f"FDR Protein: {benchmark_metrics_df.fdr_protein[idx]}<br>"
```

**Pull request merged**